modules/components/pages/outputs/couchbase.adoc

```diff
@@ -0,0 +1,272 @@
+= couchbase
+:type: output
+:status: experimental
+:categories: ["Integration"]
+
+// © 2024 Redpanda Data Inc.
+
+
+component_type_dropdown::[]
+
+
+Perform operations against a Couchbase database for each message to store or delete data.
+
+Introduced in version 4.37.0.
```

Comment:
We should conditionalize this in case we ever want to single-source it into cloud docs.

Reply:
Updated as requested.